### PR TITLE
[tensor_forest] Add total variance in the additional_data of tree leaf

### PR DESCRIPTION
**Pull request by yupbank**
To support total variance on forest level, we need to add total variance to tree level 

This pr add variance to the leaf  during export the leaf for regression task

### CODE REVIEW

**Review by nataliaponomareva — CHANGES_REQUESTED** [review]
Do you want to really add it to contrib or may be to canned directly?
It might be easier too (eg we can extend the proto instead of relying on any)

File: tensorflow/contrib/decision_trees/proto/generic_tree_model.proto [hunk 1]
```diff
@@ -46,7 +46,7 @@ message Ensemble {
     google.protobuf.Int32Value submodel_id = 2;
     repeated google.protobuf.Any additional_data = 3;
   }
-  repeated Member members = 100; // A higher id for more readable printing.
+  repeated Member members = 100;  // A higher id for more readable printing.
```
[the remark this answers is not in the document]
Comment (yupbank) [reply tensorflow/contrib/decision_trees/proto/generic_tree_model.proto]: ah... sorry, my auto lint does that... let me revert this

File: tensorflow/contrib/decision_trees/proto/generic_tree_model.proto [hunk 2]
```diff
@@ -134,14 +130,16 @@ message Leaf {
     //    vector prediction in a regression problem.
     // 5) sparse_vector holds the sparse class predictions for a classification
     //    problem with a large number of classes.
+    // 6) for regression problem, len(vector) = n_component + 1
```
[the remark this answers is not in the document]
Comment (yupbank) [reply tensorflow/contrib/decision_trees/proto/generic_tree_model.proto]: hey, so right now the option is being controlled by the `tree_path`, but we do export them into the tree.

File: tensorflow/contrib/tensor_forest/kernels/v4/leaf_model_operators.cc [hunk 3]
```diff
@@ -158,6 +159,10 @@ void RegressionLeafModelOperator::ExportModel(
         stat.weight_sum();
     leaf->mutable_vector()->add_value()->set_float_value(new_val);
   }
+  const float total_variance = TotalVariance(stat);
```
Comment (nataliaponomareva) [inline tensorflow/contrib/tensor_forest/kernels/v4/leaf_model_operators.cc]: Can you add a test to make sure it returns what u think it does

**Review by nataliaponomareva — CHANGES_REQUESTED** [review]

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**